cmd/test-external-check/main.go

```diff
@@ -53,7 +53,7 @@ func init() {
 	// Calculate check run duration from the deadline and now
 	deadline, err := strconv.Atoi(unixDeadline)
```

Comment:
I think this will break if ran and the environment variable does not exist still.  `deadline` here will equal 0, causing the `timeLimit` calculation to be 0 seconds.

Reply:
I set it to a default of 10 mins right before this

Reply:
Oh, I see that now.  Looks almost good.